manifests/federation-v2/0.0.4/federation-v2.v0.0.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,286 @@
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federation-v2.v0.0.4
+  namespace: placeholder
+spec:
+  displayName: FederationV2
+  description: |
+    Kubernetes Federation V2 namespace-scoped installation
+  version: 0.0.4
+  maturity: alpha
+  provider:
+    name: Red Hat, Inc
+  labels:
+    alm-owner-federation-v2: federation-v2
+    alm-status-descriptors: federation-v2.v0.0.4
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: federation-controller-manager
+        rules:
+        - apiGroups:
+          - clusterregistry.k8s.io
+          resources:
+          - clusters
+          verbs:
+          - "*"
+        - apiGroups:
+          - core.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - multiclusterdns.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - scheduling.federation.k8s.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - "*"
+      # TODO(font): use statefulset
+      deployments:
+      - name: federation-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: federation-controller-manager
+          template:
+            metadata:
+              labels:
+                app: federation-controller-manager
+            spec:
+              containers:
+                - name: controller-manager
+                  image: quay.io/openshift/origin-federation-controller:v4.0.0
+                  resources:
+                    limits:
+                      cpu: 100m
+                      memory: 128Mi
+                    requests:
+                      cpu: 100m
+                      memory: 64Mi
+                  command:
+                    - /root/controller-manager
+                  args:
+                    - --federation-namespace=$(FEDERATION_NAMESPACE)
+                    - --install-crds=false
+                    - --limited-scope=true
+                    - --registry-namespace=$(CLUSTER_REGISTRY_NAMESPACE)
+                  imagePullPolicy: Always
+                  env:
```

Comment:
Are either of the below equivalent to the `WATCH_NAMESPACE` env variable suggested by OLM?

Reply:
Possibly, but not in a way I'm willing to change at this point.

Reply:
Look in the vendored upstream chart for how namespaced vs. not are differentiated

Reply:
It's not an exact match for the OLM mechanics yet